lib/sensu/server/process.rb

```diff
@@ -354,7 +359,7 @@ def process_check_result(result)
               result[:check]
             end
             aggregate_check_result(result) if check[:aggregate]
-            store_check_result(client, check) do
+            store_check_result(client, check, result) do
```

Comment:
Or  perhaps `result` should be the only arg? Will have to take a look at both methods.